mock/py/mockbuild/plugins/selinux.py

```diff
@@ -114,6 +114,6 @@ def _selinuxYumIsSetoptSupported(self):
             supported = hasattr(cli.YumBaseCli, "_parseSetOpts")
             sys.path.pop(0)
             return supported
-        except SyntaxError:
+        except (SyntaxError, ModuleNotFoundError):
```

Comment:
Why this exception?... Hm looking at this function... This is 7 years old code. And it should be true on all supported platforms (RHEL6). So we do not need to discuss this and you can as well remove whole _selinuxYumIsSetoptSupported() and it will be ok.